docs/resources/gke_cluster.md

```diff
@@ -239,7 +266,30 @@ For ZonalCluster only zone information is sufficient. For Regional Cluster, both
 
 - `disable_snat` (Boolean) To use Privately Used Public IPs (PUPI) ranges, the default source NAT used for IP masquerading needs to be disabled
 - `enable_access_control_plane_global` (Boolean) With control plane global access, you can access the control plane's private endpoint from any GCP region or on-premises environment no matter what the private cluster's region is.
+- `firewall_rules` (Block List) Use FirewallRule config to specify additional firewall rules. Only Private clusters are supported.By default, tcp:9443 and tcp:22281 are opened for private cluster. (see [below for nested schema](#nestedblock--spec--config--network--access--config--firewall_rules))
 
+<a id="nestedblock--spec--config--network--access--config--firewall_rules"></a>
+### Nested Schema for `spec.config.network.access.config.firewall_rules`
+
+Optional:
+
+- `action` (String)
+- `description` (String)
+- `destination_ranges` (List of String)
+- `direction` (String)
+- `name` (String)
+- `network` (String)
+- `priority` (Number)
+- `rules` (Block List) (see [below for nested schema](#nestedblock--spec--config--network--access--config--firewall_rules--rules))
+- `source_ranges` (List of String)
```

Comment:
@akshay196-rafay `[targetTags]` is only [optional](https://github.com/RafaySystems/rafay-common/blob/c037ea94474119c861836129c7da41dbb6927515/proto/types/infraapiserver/gkecommon/gkeconfig.go#L221), by default it will populated in runtime. But if user wants he can configure as well.

Reply:
Updated.